Extensions/messaging_tweaks.js

```diff
@@ -366,6 +372,9 @@ XKit.extensions.messaging_tweaks = new Object({
 		if (XKit.extensions.messaging_tweaks.preferences.hide_send_post.value) {
 			XKit.tools.add_css(".messaging-share-post-search, .messaging-share-post-main {display:none;}", "messaging_tweaks");
 		}
+		if (XKit.extensions.messaging_tweaks.preferences.hide_online_indicators.value) {
+			$(".inbox-row.status-indicator-wrapper").hide();
```

Comment:
Also there's a few other places this should be addressed, I believe, besides the inbox dropdown. The indicator also appears on the sidebar bubbles and there's a message at the bottom of the message window:

![indicator2](https://user-images.githubusercontent.com/13437011/29464513-b0efdb66-83fb-11e7-8b99-59b917b2b62d.png)
![indicator3](https://user-images.githubusercontent.com/13437011/29464517-b36ad3d2-83fb-11e7-8149-a71b57c954dd.png)